tests/cli/output/p.txt

```diff
@@ -3,62 +3,62 @@ newlines                        = crlf
-#   6>         BRACE_OPEN[     FUNC_CLASS_DEF][  9/  1/  2/  0][1/1/0][ 140000402][0-0]         {
-#   6>            NEWLINE[               NONE][ 10/  2/  1/  0][2/2/0][         2][1-0]
-#   7>        BRACE_CLOSE[     FUNC_CLASS_DEF][  9/  1/  2/  0][1/1/0][ 140000402][0-0]         }
-#   7>            NEWLINE[               NONE][ 10/  2/  1/  0][1/1/0][         2][2-0]
-#   9>         DESTRUCTOR[               NONE][  9/  1/  2/  0][1/1/0][ 100060402][0-0]         ~
-#   9>     FUNC_CLASS_DEF[         DESTRUCTOR][ 10/  2/ 15/  0][1/1/0][     40402][0-0]          TelegramIndex
-#   9>        FPAREN_OPEN[     FUNC_CLASS_DEF][ 23/ 15/ 16/  0][1/1/0][ 100000502][0-0]                       (
-#   9>       FPAREN_CLOSE[     FUNC_CLASS_DEF][ 24/ 16/ 17/  0][1/1/0][ 100000502][0-0]                        )
-#   9>            NEWLINE[               NONE][ 25/ 17/  1/  0][1/1/0][         2][1-0]
-#  10>         BRACE_OPEN[     FUNC_CLASS_DEF][  9/  1/  2/  0][1/1/0][ 140000402][0-0]         {
-#  10>            NEWLINE[               NONE][ 10/  2/  1/  0][2/2/0][         2][1-0]
-#  11>        BRACE_CLOSE[     FUNC_CLASS_DEF][  9/  1/  2/  0][1/1/0][ 140000402][0-0]         }
-#  11>            NEWLINE[               NONE][ 10/  2/  1/  0][1/1/0][         2][2-0]
-#  13>          QUALIFIER[               NONE][  9/  1/  6/  0][1/1/0][    470402][0-0]         const
-#  13>               TYPE[               NONE][ 15/  7/ 11/  1][1/1/0][    400402][0-0]               char
-#  13>           PTR_TYPE[               NONE][ 19/ 11/ 12/  0][1/1/0][ 100400402][0-0]                   *
-#  13>          QUALIFIER[               NONE][ 21/ 13/ 18/  1][1/1/0][    410402][0-0]                     const
-#  13>               WORD[               NONE][ 27/ 19/ 27/  1][1/1/0][   1800402][0-0]                           pTelName
-#  13>          SEMICOLON[               NONE][ 35/ 27/ 28/  0][1/1/0][ 100000402][0-0]                                   ;
-#  13>            NEWLINE[               NONE][ 36/ 28/  1/  0][1/1/0][         2][1-0]
-#  14>               TYPE[               NONE][  9/  1/  9/  0][1/1/0][    470402][0-0]         unsigned
-#  14>               TYPE[               NONE][ 18/ 10/ 14/  1][1/1/0][    410402][0-0]                  long
-#  14>               WORD[               NONE][ 23/ 15/ 24/  1][1/1/0][   1800402][0-0]                       nTelIndex
-#  14>          SEMICOLON[               NONE][ 32/ 24/ 25/  0][1/1/0][ 100000402][0-0]                                ;
-#  14>            NEWLINE[               NONE][ 33/ 25/  1/  0][1/1/0][         2][1-0]
-#  15>        BRACE_CLOSE[             STRUCT][  1/  1/  2/  0][0/0/0][ 100000400][0-0] }
-#  15>          SEMICOLON[             STRUCT][  2/  2/  3/  0][0/0/0][ 100000000][0-0]  ;
-#  15>            NEWLINE[               NONE][  3/  3/  1/  0][0/0/0][         0][2-0]
+# Line                Tag              Parent          Columns Br/Lvl/pp     Flag   Nl  blockNu Text

```

Comment:
I just noticed you are not capitalizing here, while the other column headers are all capitalized. This also makes me wonder if the "Nu" is really needed here; just "Block" might be better. (Note: this is prose and should not be subject to the same style rules as code.)

Off topic for this PR, but now that I'm thinking about it, "pp" being all lower case seems a little odd as well :wink:.